dapp/src/components/buySell/ApproveSwap.js

```diff
@@ -99,19 +115,6 @@ const ApproveSwap = ({
     stableCoinToApprove,
     isMobile,
   }) => {
-    const capitalized = selectedSwap.name.charAt(0).toUpperCase()
-    const noncapitalized =
-      selectedSwap.name === 'uniswapV2'
-        ? selectedSwap.name.slice(1, 7)
-        : selectedSwap.name.slice(1)
-    const origin =
-      selectedSwap.name === 'flipper' || selectedSwap.name === 'vault'
-        ? 'the '
-        : ''
-    const vault = selectedSwap.name === 'vault' ? 'Origin ' : ''
-    const coin = stableCoinToApprove.toUpperCase()
-    const route = `${origin} ${vault} ${capitalized}${noncapitalized} to use your ${coin}`
-    const routeMobile = `${origin} ${vault} ${capitalized}${noncapitalized}`
```

Comment:
Above bit of code was hard to understand for me. Which means it is hard to maintain whenever we need to revisit it and do any modifications. Maintainability of the code is important, so we can move fast and adjust stuff fast when/if needed.